test/column_variables.jl

```diff
@@ -0,0 +1,45 @@
+@testset "ColumnVariables initialisation" begin
+
+    # no number format provided
+    # column = ColumnVariables(nlev=8)
+    # @test eltype(column.temp) == Float64
+
+    @testset for NF in (Float16,Float32,Float64)
```

Comment:
This test includes `Float16` but the following don't.

Reply:
Problem is that we still can't `initialize_speedy(Float16)` as I haven't yet included GenericFFT.jl, see #31. So Float16 testing wherever possible, but FFTW only allows for Float32/64.